personal_profile/tests.py

```diff
@@ -68,3 +64,17 @@ def test_profile_is_deleted_when_user_is_deleted(self, profile_1, user_1):
         test_user = User.objects.filter(username='user_1').first()
         test_user.delete()
         assert test_profile not in PersonalProfile.objects.all()
+
+
+@pytest.mark.django_db
+class TestProfileDetailView:
+    def test_detail_view_page_entrypoint(self, profile_1, user_1, client):
+        # Testing to see if a valid user id gets a valid detail view page
+        response = client.get(PROFILE_DETAIL_URL + str(profile_1.id) + '/')
+        assert response.status_code == 200
```

Comment:
Is there a way to verify valid user gets valid detail view from  response? 
Maybe assert returned template? 
personalprofile_detail.html

You could also verify the returned object from DetailedView,
you should be able to read this in response
personalprofile

Reply:
Added the test of the template.
Regarding the 2nd suggestion- tried but couldn't understand how to access the DetaileView object from the response.

Reply:
Please look at template
It is reading the following data
personalprofile.user.username and other files, please review UI talk, and try to Can you receive the personalprofile from response.context?